Common/ac/game_version.h

```diff
@@ -134,7 +137,8 @@ enum GameDataVersion
     kGameVersion_341            = 48,
     kGameVersion_341_2          = 49,
     kGameVersion_350            = 50,
-    kGameVersion_Current        = kGameVersion_350
+    kGameVersion_350_1          = 51,
```

Comment:
Version constant should be kGameVersion_351, because this feature will be added after 3.5.0 already. XXX_N naming  was used previously when there were additional format changes after first public (alpha or beta) releases.